src/inspector_agent.h

```diff
@@ -33,6 +35,9 @@ class Agent {
   // Stop the inspector agent
   void Stop();
 
+  void ContextCreated(const v8_inspector::V8ContextInfo& info);
```

Comment:
There's still some churn in inspector API (e.g. https://github.com/nodejs/node/pull/9028 is quite major) so I feel like it would be safer to only rely on v8 namespace.